### DISCUSSION TITLE
change root user

### DISCUSSION THREAD

**Discussion by mohamadchoker** [Homebrew/discussions#3153] [Linux]
Hello, 

I ran brew services start nginx as root by mistake so I removed Nginx and all its files.
now every time I reinstall nginx the default user is root and i need to change it to the default user.

`nginx      none    root`

**Comment by SMillerDev** [comment]
I'm not sure what you mean by
> every time I reinstall nginx the default user is root and i need to change it to the default user.

Brew won't run as a root user so there is no way to install nginx as root with brew.

If you mean brew complains it can't remove the old nginx install, you'll have to do `sudo chown -R $(whoami) /usr/local/*` or `sudo chown -R $(whoami) /opt/homebrew/*` to resolve that.

**Reply by mohamadchoker** [reply]
![Screen Shot 2022-04-06 at 3 00 30 PM](https://user-images.githubusercontent.com/11899428/161970043-7229e1f4-1719-4e6d-903d-e209f988ff07.png)

**Reply by SMillerDev** [reply]
You can try `sudo brew services stop nginx`

**Reply by booster** [reply]
Starting as root will cause an plist in /Library/LaunchDaemons. Remove it and try to run brew services start nginx, and the correct user should now be listed under the brew services list of running/failing services.
